lib/config/presets/common.ts

```diff
@@ -23,6 +23,7 @@ export const removedPresets: Record<string, string | null> = {
   'default:app': 'config:js-app',
   'default:js-app': 'config:js-app',
   'default:library': 'config:js-lib',
+  'default:onlyNpm': null,
```

Comment:
Please double check if this is the correct preset name.

Is there anything else I need to do to properly remove/migrate the `onlyNpm` preset?

The Creating/editing Renovate preset docs do **not** explain how to **remove** a preset:

https://github.com/renovatebot/renovate/blob/main/docs/development/creating-editing-renovate-presets.md